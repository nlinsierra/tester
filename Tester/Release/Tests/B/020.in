3H JH AD QD 5S 8C AS 3S JD QC AC KC 2D 6D 4D KD 4S 8D TC 5D 9S 7D 4C 5C 2H 7S 9C 2S 7C 4H 3C 6H 9H JS QH AH QS JC 3D TS 8S 9D 6S KS KH TH 2C 7H 6C
