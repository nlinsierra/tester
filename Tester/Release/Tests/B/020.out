3
TD 5H 8H
